packages/playground/src/dashboard/components/add_ip.vue

```diff
@@ -201,6 +212,12 @@ export default {
           message: "IP is not public.",
         };
       }
+      const ips = await gqlClient.publicIps({ ip: true }, { where: { ip_eq: publicIP.value } });
+      if (ips.length > 0) {
+        return {
+          message: "IP exists",
+        };
+      }
```

Comment:
We use the same function implementation twice. Can we extract it outside the block and call it in both places?